source/plugin/configuration/loaders.rst

```diff
@@ -16,6 +17,10 @@ configurations, giving you the option of hard-coding default values or loading f
 Getting your Loader
 ~~~~~~~~~~~~~~~~~~~
 
+..note::
```

Comment:
Missing a space between `..` and `note::`, see https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/a914c77/plugin/configuration/loaders.html